clients/fides-js/src/fides-ext-gpp.ts

```diff
@@ -176,7 +176,12 @@ const initializeGppCmpApi = () => {
     //    - User has no existing preferences (either in cookie, fides_string, or mapped to notices)
     if (
       !fidesString &&
-      (!shouldResurfaceConsent(experience, event.detail, savedConsent) ||
+      (!shouldResurfaceBanner(
+        experience,
+        event.detail,
+        savedConsent,
+        options,
+      ) ||
```

Comment:
In the case where fidesString already exists, I still see that `window.Fides.fides_string` being overwritten with just the GPP string section. E.g. add a `fides_string` cookie of `,,,eyJhbmFseXRpY3MiOjEsImZ1bmN0aW9uYWwiOjEsImVzc2VudGlhbCI6MCwic2FsZXNfc2hhcmluZ190YXJnZXRlZF9hZHZlcnRpc2luZ19ncHBfdXNfbmF0aW9uYWwiOjB9`, the load in the GPP version of fides.js. See `window.Fides.fides_string` does not include `nc` string, e.g. `',,DBABLA~BAAUAAAAAABk.QA'`